opm/autodiff/SimulatorFullyImplicitBlackoilEbos.hpp

```diff
@@ -295,6 +295,9 @@ class SimulatorFullyImplicitBlackoilEbos
             const double nextstep = adaptiveTimeStepping ? adaptiveTimeStepping->suggestedNextStep() : -1.0;
             ebosSimulator_.problem().setNextTimeStepSize(nextstep);
             ebosSimulator_.problem().writeOutput();
+            ebosSimulator_.problem().applyActions(timer.currentStepNum(),
+                                                  ebosSimulator_.vanguard().schedule(),
+                                                  ebosSimulator_.vanguard().summaryState());
             report.output_write_time += perfTimer.stop();
```

Comment:
I would prefer that the applyAction(...) call is moved to endEpisode(...) in eclproblem.hh rather than put here.

Reply:
Sure